nft-simple/src/internal.rs

```diff
@@ -43,14 +43,14 @@ pub(crate) fn deposit_refund(storage_used: u64) {
     }
 }
 
-pub(crate) fn bytes_for_approved_account_id(account_id: &AccountId) -> u64 {
+pub(crate) fn bytes_for_approved_account_id(hm: (&AccountId, &u64)) -> u64 {
     // The extra 4 bytes are coming from Borsh serialization to store the length of the string.
-    account_id.len() as u64 + 4
+    hm.0.len() as u64 + 4
 }
```

Comment:
This needs more now. Since we store u64 as well.

Reply:
Good catch, can you help me figure this one out? Not sure where to start on determining what should be there instead of `4`

Reply:
it should be `(4 + size_of::<u64>())`